config/kafka-instance-types-configuration.yaml

```diff
@@ -1,6 +1,6 @@
-# This configuration defines a list of supported Kafka instance types and their available sizes for KAS Fleet Manager.
+# This configuration defines a list of supported Kafka instance types and their available sizes for KAS Fleet Manager. 
 # Modify this file to add additional Kafka instance types and sizes or change the default values listed below.
-#
+# 
 # The following properties must be defined for each Kafka instance type:
 #   - id: Identifier for the Kafka instance type. Each instance type name should be unique.
```

Comment:
Let's add the `display_name` description in the docs here.

Reply:
done